api-documentation.md

```diff
@@ -61,7 +60,7 @@ aaa@test.pl;3.5;2;5;1;https://megak.pl`
   "error": "Bad Request",<br/>
   "statusCode": 400<br/>
   }<br/>
-- res — json: {ok: true} | patrz wyżej
+- res — json: {type `FailedEmails` (będzie pusty), type `SuccesfulEmails` (tablica z jednym elementem)}
```

Comment:
trzeba by dodać jaki to element - jeśli pasuje do typu/interfejsu to wystarczy nazwę tegoż